servatrice/servatrice.ini.example

```diff
@@ -99,6 +99,15 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Enable forgot password functionality
+enableForgotPassword=true
+
+; Validate users inputed email matches email on account before allowing forgot password reset
+forgotpasswordemailrequired=true
+
+; Validate users clientid matches clientid on account before allowing forgot password reset
+forgotpasswordclientidrequired=true
```

Comment:
Checked may be a better term to use and could resolve the confusion over the word "required".  As far as the idea on sending an email, the clientid should not change.  Even if the id does change I wouldn't want the user to be able to change there password in that case if I have the check for clientid as a required, otherwise i as an admin would just disable the check for client id option.

Plus there is always the elephant in the room about sending emails, what happens if I dont have an email server to use to send the emails?